CONTRIBUTING.md

```diff
@@ -0,0 +1,165 @@
+# DRAFT Contributing Guidelines
+
+Thank you for considering a contribution to the club-40 group project! To get you started, here are a few guidelines we would like you to follow:
+
+* [Issues and Bugs](#issue)
+* [Feature Requests](#feature)
+
+## <a name="question"></a> Got a Question?
```

Comment:
Does `<a name="">` do the same thing as `<a id="question" href="#"></a>`?

Not a blocker, just curious. Otherwise looks good to me.

Reply:
Yep. same functionality. I just tested it.